cs4360/ormconfig.json

```diff
@@ -1,14 +1,19 @@
 {
```

Comment:
Did we ever figure out if we need both of these? Both this file and the configuration in the API Module?